### PR TITLE
Rework JSDoc comments

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: src/acl.js [hunk 1]
```diff
@@ -212,23 +221,36 @@ Acl.prototype.writers = {};
...
- * @param {object} options - Configuration object.
- * @param {string} options.entity - Whose permissions will be added.
- * @param {string} options.role - Permissions allowed for the defined entity.
- *     See {module:storage#acl}.
- * @param {int=} options.generation - **File Objects Only** Select a specific
+ * @param {object} options Configuration options.
+ * @param {string} options.entity Whose permissions will be added.
+ * @param {string} options.role Permissions allowed for the defined entity.
+ *     See {@link https://cloud.google.com/storage/docs/access-control Access Control}.
+ * @param {numer} [options.generation] **File Objects Only** Select a specific
```
Comment (lukesneeringer) [inline src/acl.js]: `number`

File: src/acl.js [hunk 2]
```diff
@@ -254,6 +276,24 @@ util.inherits(Acl, AclRoleAccessorMethods);
...
  *   var apiResponse = data[1];
  * });
+ *
+ * @example <caption>include:storage/acl.js</caption>
+ * region_tag:storage_add_file_owner
+ * organization:GoogleCloudPlatform
+ * repository:nodejs-docs-samples
+ * Example of adding an owner to a file:
+ *
+ * @example <caption>include:storage/acl.js</caption>
```
Comment (lukesneeringer) [inline src/acl.js]: Seems like these should be:

```
@example <caption>include:samples/acl.js</caption>
region_tag:storage_add_file_owner
organization:googleapis
repository:nodejs-storage
```

Thoughts?

P. S. Are `organization` and `repository` necessary if they point to the same repo?

Comment (jmdobry) [reply src/acl.js]: Sent you a response

Comment (jmdobry) [reply src/acl.js]: The need for those was temporary, I've removed them

**Comment by jmdobry** [comment]
Let's hold off on merging until I'm actually ready to publish the docs.

**Comment by lukesneeringer** [comment]
Ack.

**Pull request merged**